doc/tech/approach.md

```diff
@@ -3,97 +3,230 @@
-1. Provide ability for any number of Operator-specific fabrics to power CNTT cloud infrastructure
-1. Enable RC's ability to realize mandated OVP qualification deliverables
-1. Unambiguously document the responsibilities of each CNTT constituency
-1. In cases where a VNF/CNF require HW layer resources it should be under the control of the Virtualization Layer
-1. It is important that the HW Infrastructure Manager, each VIM and each VNF/CNF could be managed by separate organizations
-1. Drive the industry towards convergence on ABIs supporting Cloud-Native implementations for SmartNICs
-
-> _**Editor's Note:** Consider moving objectives to a table_
+### 5.1.2 Networking Principles & Strategy
+
+This section introduces some networking related principles that should be followed during development and definition of Networking scopes for Reference Model, Reference Architecture 01 (OpenStack), Reference Architecture 02 (K8s) and down to Reference Implementations and Reference Conformance test suites under CNTT.
+
+•	**Abstraction**: A standardized layer between Virtualisation Layer domains and the Networking Hardware Layer resource pool hiding the implementation details of the Networking Hardware.
+> **Note:** Abstraction may be deployed in many different ways e.g. depending on target use case requirements, workload characteristics, different algorithm implementions of pipeline stages and available platforms. This includes Network Functions running on server nodes with or without programmable hardware acceleration, or networking support functions running on programmable network switches in the network.
+
+•	**Agnosticity** : Define Network Fabric concepts and models that can carry any type of traffic in terms of:
+-	Control, User and Management traffic types
+-	Acceleration technologies supporting multiple types of Infrastructure deployments and Network Function workloads 
+
+•	**Automation**: Enable full automation, from Physical Fabric installation and provisioning to automation of VNF/CNF onboarding.
+
+•	**Openness**: All Networking is based on open source or standardized APIs (NBI and SBI) and should enable integration of Open Source networking components (e.g. SDN controllers).
+
+•	**Programmability**: Should enable a programmable forwarding plane controlled from a separately deployed control plane that abstracts into standardized functions and interfaces.
+
+•	**Scalability**: Should be scalable to handle all traffic traverse North-South and East-West enabling small up to large deployments.
+
+•	**Workload diversity**: Should be capable to handle any VNF , CNF and BareMetal workloads.
+
+•	**Telco Grade**: Should support deployments with Telco Grade characteristics requirements.
+
+•	**Future proof**: Should support known and emerging acceleration technologies e.g. SmartNICs, FPGAs and Programmable Switches. 
+
 
 <a name="5.1.3"></a>
-### 5.1.3 Networking Requirements
+### 5.1.3 Network Layering and Concepts
```

Comment:
Can we move this to a separate PR please?
The title for this is to define the scope.

Reply:
Will provide detailed comments when we have agreed on the scope and some principles there.

A general comment: The layering description goes far beyond the networking domain. If the NFVI is layered in the proposed way, also compute and storage domain are affected. We should not hide that in networking specific chapters if we want to require such layers.

Reply:
@ulikleber I agree completely that this Layering goes beyond networking, but I don´t agree that someone is trying to hide it!
Being a Networking chapter from a Networking Focus Group as the first step of handling truly shared resources, it is a very logical first step that must be managed at least on the Networking Underlay that must be shared.
All other "shared HW resources" as Compute, Storage and possibly Accelerators HW resources are really not needed to be simultaneously shared, but rather "semi permanently allocated" to a workload at any point in time.

Reply:
@TFredberg sorry for using "hide". What I mean is that such a layering - since it affects also compute and storage - 
needs to be discussed as a general architectural principle by all experts, not only in a networking WS.
ETSI doesn't have such a layering. So we have to explain it also as an additional concept, not like it would be part of the NFV reference architecture.